docs/training/mobile.rst

```diff
@@ -0,0 +1,209 @@
+
+==================================
+Aloha Mobile Training & Evaluation
+==================================
+
+.. note::
+
+   **ACT++** is only supported with **Aloha 1.0**.
+   Ensure that you are using compatible software versions to avoid any issues during training or evaluation.
+
+Install Dependencies
+^^^^^^^^^^^^^^^^^^^^
+
+Install the necessary dependencies inside your containerized environment:
+
+.. code-block:: bash
+
+    $ pip install dm_control==1.0.14
+    $ pip install einops
+    $ pip install h5py
+    $ pip install ipython
+    $ pip install matplotlib
+    $ pip install modern_robotics
+    $ pip install mujoco==2.3.7
+    $ pip install opencv-python
+    $ pip install packaging
+    $ pip install pexpect
+    $ pip install pyquaternion
+    $ pip install pyyaml
+    $ pip install rospkg
+    $ pip install torch
+    $ pip install torchvision
+    $ pip install transforms3d
+    $ pip install wandb
+
+Clone Robomimic
+^^^^^^^^^^^^^^^
+
+.. code-block:: bash
+
+    $ cd ~
+    $ git clone -b r2d2 https://github.com/ARISE-Initiative/robomimic.git
+    $ cd robomimic
+    $ pip install -e .
+
+Clone ACT++ Repository
+^^^^^^^^^^^^^^^^^^^^^^
+
+
```

Comment:
Leave only one line of whitespace between blocks of text. Do this everywhere

```suggestion
Clone ACT++ Repository
^^^^^^^^^^^^^^^^^^^^^^

```

Reply:
Okay